lib/usd/ui/views/TreeItem.h

```diff
@@ -0,0 +1,104 @@
+//
+// Copyright 2019 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#pragma once
+
+#include "../api.h"
+
+#include <QtGui/QStandardItem>
+
+#include <pxr/usd/usd/prim.h>
+
+PXR_NAMESPACE_USING_DIRECTIVE
+
+MAYAUSD_NS_DEF {
+
+/**
+ * \brief Item representing a node used to build a Qt TreeModel.
+ * \remarks This item is intended to hold references to USD Prims in the future, so additional information can be
+ * displayed to the User when interacting with Tree content.
+ */
+class MAYAUSD_UI_PUBLIC TreeItem : public QStandardItem
+{
```

Comment:
Note: there is no Q_OBJECT here as QStandardItem does not derive from QObject. So there cannot be signals/slots in this class.